common/source/docs/common-frsky-passthrough.rst

```diff
@@ -0,0 +1,62 @@
+.. _common-frsky-passthrough:
+
+====================================
+Passthrough FrSky Telemetry Protocol
+====================================
+
+Overview
+========
+
+Passthrough FrSky Telemetry is the latest protcol available in ArduPilot and it allows the transmission of raw data directly to OpenTX without any preprocessing by OpenTX and without the need for discovery. The passthrough telemetry protocol is optimized specifically for the FrSky datalink bandwidth and enhancements include having multiple data elements in a single telemetry packet and using floating point representation (e.g. we don't really need centimeter precision altitude to be displayed on screen when flying at 100m). This results in an improved data refresh rate such that information such as attitude (roll, pitch, yaw) is displayed without any perceptible lag.
```

Comment:
Let's continue this on the issue thread since it doesn't belong here, but again: Why do you need to run discovery for more than the required few seconds necessary to discover the sensors?